coverage.py

```diff
@@ -165,7 +165,6 @@ def run_on_specific_identifiers(self, identifiers):
             transient_failures += t
             persistent_failures += p
             records += r
-            self._db.commit()
```

Comment:
No need to do this because it just happened in the call to `process_batch_and_handle_results`.